### DISCUSSION TITLE
Persistent Characters

### DISCUSSION THREAD

**Discussion by GhostTownCowboy** [lllyasviel/Fooocus#1178] [General]
I'm sorry if I missed this elsewhere, is there a way to establish and use persistant characters in Fooocus? I tried with the seed but the faces are all just different enough to be obvious.

**Comment by mel870** [comment]
The following thread has some tips on using Faceswap which could be a solution: 

https://github.com/lllyasviel/Fooocus/discussions/557

**Comment by botzero-net** [comment]
Some options:
- Train and use a LoRA model on your character(s)--look up "Taco Dog" on Civit for an example of a SDXL-compatible LoRA that works with Fooocus
- Use a combination of image prompts together, including faceswap--check my pinned post on x(dot)com(slash)botzero_net for help
